occu-med-portal/src/index.css

```
@@ -82,6 +82,9 @@
   }
   body {
     @apply font-sans bg-background text-foreground antialiased;
+    -webkit-font-smoothing: antialiased;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)